discojuice exception

this started after one of the recent changes
```
Caused by: java.lang.NullPointerException
        at java.util.Hashtable.put(Hashtable.java:459)
        at java.util.Properties.setProperty(Properties.java:166)
        at java.lang.System.setProperty(System.java:793)
        at cz.cuni.mff.ufal.DiscoJuiceFeeds.writeFeedsFile(DiscoJuiceFeeds.java:147)
        at cz.cuni.mff.ufal.DiscoJuiceFeeds.refreshFeedsFile(DiscoJuiceFeeds.java:117)
        at cz.cuni.mff.ufal.DiscoJuiceFeeds.act(DiscoJuiceFeeds.java:83)
        at org.apache.cocoon.sitemap.impl.DefaultExecutor.invokeAction(DefaultExecutor.java:55)
        at org.apache.cocoon.components.treeprocessor.sitemap.ActTypeNode.invoke(ActTypeNode.java:105)
        ... 61 more
```

--- a/dspace-xmlui/src/main/java/cz/cuni/mff/ufal/DiscoJuiceFeeds.java
+++ b/dspace-xmlui/src/main/java/cz/cuni/mff/ufal/DiscoJuiceFeeds.java
@@ -144,7 +144,9 @@ public class DiscoJuiceFeeds extends AbstractAction {
             log.error("Failed to obtain/parse "+shibDiscoFeedUrl.toString() + "\nCheck timeouts, redirects, shibboleth config.\n" + e);
             throw e; //Don't continue
         }finally {
-            System.setProperty("jsse.enableSNIExtension", old_value);
+        	//true is the default http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html
+        	old_value = (old_value == null) ? "true" : old_value;
+        	System.setProperty("jsse.enableSNIExtension", old_value);
         }
 
         //String[] feeds = {"edugain", "cesnet"};

--- a/dspace-xmlui/src/main/java/cz/cuni/mff/ufal/PiwikStatisticsReader.java
+++ b/dspace-xmlui/src/main/java/cz/cuni/mff/ufal/PiwikStatisticsReader.java
@@ -149,6 +149,8 @@ public class PiwikStatisticsReader extends AbstractReader {
             }
             in.close();
         }finally {
+        	//true is the default http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html
+        	old_value = (old_value == null) ? "true" : old_value;
             System.setProperty("jsse.enableSNIExtension", old_value);
         }
 		return output.toString();

